core/evancore.vue.libs/src/component.ts

```diff
@@ -106,6 +105,6 @@ export default class EvanComponent extends Vue {
    * @return     {any}  bcc runtime
    */
   getRuntime(): Runtime {
-    return this.$store.state.runtime || bccHelper.getCoreRuntime();
+    return this.$store.state.runtime;
```

Comment:
according too Vuex $store states should be plain Objects, this would actually exclude the possibility to keep the runtime in the $store. Let's keep this in mind, and discuss it.